goal_src/jak1/pc/pckernel-common.gc

```diff
@@ -256,7 +258,9 @@
   `(begin
     (cpad-clear! 0 r1)
     (logxor! ,cheats-var (pc-cheats ,cheat))
-    (cheats-sound-play (logtest? ,cheats-var (pc-cheats ,cheat)))
+    (if (-> *pc-settings* speedrunner-mode?)
+    (sound-play "caught-eel")
```

Comment:
This sound effect is not in Jak 2! (what's the reason for this change anyway?) Also, fix the indenting please.

Reply:
The reasoning for the change is in the commit 

```Add bad sound feedback if a cheat is attempted to toggle with speedrunner mode on

Before this it just didnt turn on but gave zero feedback so it was kind of bad...```

Then there is a video that is supplied as well. Do you have any suggestions for a sound that is present in both games?

Reply:
Oh, well it wasn't apparent that speedrun mode locked out cheats. Why not just *not set* the cheat variable instead of this, then?

Reply:
This code still lets the cheat code go through. If you input the cheat for blue eco, you will still get blue eco even with speedrun mode enabled.
![image](https://github.com/open-goal/jak-project/assets/7569514/158ed342-48d8-47bd-bc5f-88ae202b6178)